pyglmnet/pyglmnet.py

```diff
@@ -851,6 +852,14 @@ def fit(self, X, y):
                 if self.fit_intercept:
                     ActiveSet[0] = 1.
 
+            # Convergence by relative parameter change tolerance
```

Comment:
This criterion on its own is sufficient for all tests to pass

Reply:
Do the convergence warnings go away once we switch to this criteria?

Reply:
yes